tests/unit/zhmcclient/test_notification.py

```diff
@@ -23,14 +23,38 @@
 
 import json
 import threading
+from collections import namedtuple
 from mock import patch
 import six
 import pytest
+import stomp
 
 from zhmcclient._notification import NotificationReceiver
 from zhmcclient._exceptions import SubscriptionNotFound
 
 
+def create_event_args(headers, message):
+    """
+    Transform headers, message to event method parameters.
+    This is the inverse of _NotificationListener.get_headers_message().
+    """
+    # Indicator whether stomp uses Frame objects for the event listener
+    # methods. stomp.py introduced this in version 7.0.0, but since it
+    # changed its __version__ attribute from tuple to string in version
```

Comment:
Out of curiosity: isn't using a tuple for version "better" than a flat string?

Reply:
By convention, `__version__` contains the string, and `__version_info__` or `version_info` the tuple.
https://stackoverflow.com/a/459185/1424462
https://peps.python.org/pep-0396/